### PR TITLE
EOS-18545: Move the common config/code from the sensors/message_handl…

### PR DESCRIPTION
**Pull request by gargpalak048**
…ers to the Utility class.

## Problem Statement
<pre>
  <code>
    Story Ref (if any):
    Please add Problem statement here...
  </code>
</pre>
## Problem Description
<pre>
  <code>
     optimize the code to fetch config values across the SSPL
  </code>
</pre>
## Solution
<pre>
  <code>
   Mapped the config keys, values in conf_utils.py and all the files will fetch configs from there. 
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
    - [ ] No
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
    Please describe the tests that you ran to verify your changes.
  </code>
</pre>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: low-level/files/opt/seagate/sspl/setup/sspl_prepare.py [hunk 1]
```diff
@@ -47,61 +48,40 @@ def validate(self):
...
-            "server_node>%s>type" % machine_id)
-        cluster_id = Utility.get_config_value(PRVSNR_CONFIG_INDEX,
-            "server_node>%s>cluster_id" % machine_id)
-        if node_type.lower() not in ["virtual", "vm"]:
-            bmc_ip = Utility.get_config_value(PRVSNR_CONFIG_INDEX,
-                "server_node>%s>bmc>ip" % machine_id)
-            bmc_user = Utility.get_config_value(PRVSNR_CONFIG_INDEX,
-                "server_node>%s>bmc>user" % machine_id)
-            secret = Utility.get_config_value(PRVSNR_CONFIG_INDEX,
-                "server_node>%s>bmc>secret" % machine_id)
```
Comment (mariyappanp) [inline low-level/files/opt/seagate/sspl/setup/sspl_prepare.py:66]: `Utility.get_config_value` method validates and throw if value is None or key is not found in config.
Where did you move the validation? At each mini provisioner stage, its required to validate the input configs.
```
[root@ssc-vm-2217 ~]# python3
Python 3.6.8 (default, Nov 16 2020, 16:55:22)
[GCC 4.8.5 20150623 (Red Hat 4.8.5-44)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>>
>>> from cortx.utils.conf_store import Conf
>>> Conf.load("conf", "yaml:///etc/sspl.conf")
>>> Conf.get("conf", "server_node>dummy")
>>>
```
Conf won't throw exception on invalid key or null value for a mandatory key.

File: low-level/framework/utils/conf_utils.py [hunk 2]
```diff
@@ -167,3 +170,49 @@
 BMC_IP_KEY = "server_node>%s>bmc>ip" % MACHINE_ID
```
Comment (mariyappanp) [inline low-level/framework/utils/conf_utils.py:170]: > CNTRLR_PASSWD_KEY = "storage_enclosure>%s>controller>password" % ENCLOSURE
Please adapt latest changes on config keys.
`password` is now `secret`.

File: low-level/framework/base/sspl_constants.py [hunk 3]
```diff
@@ -74,6 +74,11 @@
 SSPL_CONFIG_INDEX = "SSPL"
 SSPL_TEST_CONFIG_INDEX = "SSPL_TEST"
 CONFIG_SPEC_TYPE = "yaml"
+DEFAULT_DC = "DC01"
+DEFAULT_RACK = "RC01"
+DEFAULT_SN = "SN01"
+DEFAULT_CLUSTER = "CC01"
```
Comment (mariyappanp) [inline low-level/framework/base/sspl_constants.py:80]: When config API is not available to serve input data for sspl, why should we proceed with default value?

Ex. Encrypted password(secret) is based on cluster_id. With default cluster_id enclosure sensor won't get login to the HW enclosure setup as decrypted password would be wrong.

File: low-level/files/opt/seagate/sspl/setup/sspl_prepare.py [hunk 4]
```diff
@@ -27,6 +27,7 @@
 from cortx.utils.validator.v_network import NetworkV
 from files.opt.seagate.sspl.setup.setup_error import SetupError
 from framework.utils.utility import Utility
+from framework.utils.conf_utils import fetch_config
```
Comment (mariyappanp) [inline low-level/files/opt/seagate/sspl/setup/sspl_prepare.py:30]: This became unused as you removed its reference call.
Please check and remove it if nowhere used inside this module.

File: low-level/files/opt/seagate/sspl/setup/sspl_prepare.py [hunk 5]
```diff
@@ -27,6 +27,7 @@
 from cortx.utils.validator.v_network import NetworkV
 from files.opt.seagate.sspl.setup.setup_error import SetupError
 from framework.utils.utility import Utility
```
Comment (mariyappanp) [inline low-level/files/opt/seagate/sspl/setup/sspl_prepare.py:29]: This has become unused as you removed its reference call.
Please check and remove it if nowhere used inside this module.

File: low-level/files/opt/seagate/sspl/setup/sspl_prepare.py (hunk 1 above)
Comment (gargpalak048) [reply low-level/files/opt/seagate/sspl/setup/sspl_prepare.py]: Agree. 
the setup scripts should not apply these changes anyway, only service files will read config from conf_utils.py.
Reverted the changes and keeping the prepare & setup scripts as original.

**Pull request closed without merging**